apps/web/components/booking/CancelBooking.tsx

```diff
@@ -95,10 +100,11 @@ export default function CancelBooking(props: Props) {
                   const res = await fetch("/api/cancel", {
                     body: JSON.stringify({
                       uid: booking?.uid,
-                      cancellationReason: cancellationReason,
+                      cancellationReason,
                       allRemainingBookings,
-                      // @NOTE: very important this shouldn't cancel with number ID use uid instead
```

Comment:
Can you bring back this comment?

Reply:
ok

Reply:
https://github.com/calcom/cal.com/pull/15684 completed here